src/web/tools/KeyEventManager.ts

```diff
@@ -0,0 +1,81 @@
+import { AdaptedEvent, EventTypes } from '../interfaces';
+import EventManager from './EventManager';
+import { PointerType } from '../../PointerType';
+
+export default class KeyEventManager extends EventManager<HTMLElement> {
```

Comment:
Since we operate on `KeyboardEvents`, I think it would be better to call it `KeyboardEventManager`

```suggestion
export default class KeyboardEventManager extends EventManager<HTMLElement> {
```

Reply:
Renamed as of [76bf761](https://github.com/software-mansion/react-native-gesture-handler/pull/3035/commits/76bf76174eb4f1bf186f6ba07b3a368bf6925062)